.github/workflows/build.yml

```diff
@@ -0,0 +1,200 @@
+name: Build the app for Android and iOS
+
+  # CI for MapSwipe app
+  #
+  # This runs on each push, and does different things depending on
+  # the branch and tags of the current commit.
+  #
+  # Secrets come from environment variables from github.
+  # Files containing secrets are base64 encoded, and their content
+  # loaded from a secret env var.
+  # For binary files, this does not seem to work, so we use gpg to hide them.
+  # (See comments below)
+  #
+  # Releases:
+  # They should always be created using the script in ./scripts/version.sh
+  # which will run a few checks and tag commits appropriately.
+
+on: [push]
+
+jobs:
+
+  check-syntax:
+    name: Validate code format and types before building
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '16.1.0'
+      - name: Install javascript dependencies
+        uses: bahmutov/npm-install@v1
+      - name: Lint the code
+        run: yarn lint
+      - name: Check typing of code
+        run: yarn flow
+      - name: Lint JSON translation files
+        run: yarn lintJson
+
+  release-android:
+    name: Build and release Android app
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '16.1.0'
+      - name: Install javascript packages
+        uses: bahmutov/npm-install@v1
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.7.3'
+      - name: Decode sentry.properties
+        env:
+          SENTRY_PROPERTIES: ${{ secrets.FILE_SENTRY_PROPERTIES }}
+        run: echo "$SENTRY_PROPERTIES" | base64 -d > ./android/sentry.properties
+      - name: Decode development google-services.json
+        env:
+          FILE_ANDROID_GOOGLE_SERVICES_DEV: ${{ secrets.FILE_ANDROID_GOOGLE_SERVICES_DEV }}
+        run: echo "$FILE_ANDROID_GOOGLE_SERVICES_DEV" | base64 -d > ./android/app/src/dev/google-services.json
+      - name: Decode production google-services.json
+        env:
+          FILE_ANDROID_GOOGLE_SERVICES_PRODUCTION: ${{ secrets.FILE_ANDROID_GOOGLE_SERVICES_PRODUCTION }}
+        run: mkdir -p ./android/app/src/production ; echo "$FILE_ANDROID_GOOGLE_SERVICES_PRODUCTION" | base64 -d > ./android/app/src/production/google-services.json
```

Comment:
here you first create the directory. For the dev service account you don't do this.  Is there a reason why you have do it here?

Reply:
The `dev` directory already exists because there are files in it (icons/splash screen). I'm not sure why there are no equivalent files in the production folder, but that's how it is for now, and it works, so I won't wake up the sleepy bear :bear: